src/components/Avatar/Avatar.component.tsx

```diff
@@ -348,28 +355,17 @@ const Avatar: FC<AvatarProps> = ({
       />
       {AvatarModel}
       {children}
-      {shadows && <ContactShadows position={[0, 0, 0]} opacity={2} scale={10} blur={1.0} far={1.0} />}
+      {shadows && (
+        <ContactShadows opacity={effects?.ambientOcclusion ? 1 : 2} scale={4} blur={2} far={1.0} resolution={256} />
```

Comment:
ao and shadows don't behave well together, change the oopacity values depending on if its enabled or not.